src/randoop/NoExceptionCheck.java

```diff
@@ -12,9 +12,11 @@
   
   // Indicates which statement is expected to return normally. 
   private final int statementIdx;
+  private String exceptionName;
   
-  public NoExceptionCheck(int statementIdx) {
+  public NoExceptionCheck(int statementIdx, String exceptionName) {
```

Comment:
It's a bit surprising that the documentation says "Checks that no exception is thrown by a given statement" but the constructor takes a specific exception name as an argument.  Can you clarify the semantics or intended use?